Documentation/building/linux-instructions.md

```diff
@@ -36,7 +36,9 @@ ellismg@linux:~$ sudo apt-get update
 
 Then install the packages you need:
 
-`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext`
+```bash
+ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext
+```
```

Comment:
I would imagine that leaving those in were a conscious choice to show the working directory later in the guide (for example, [line 127 of `linux-instructions.md`](https://github.com/cubrr/coreclr/commit/b246686a85f59d4a3ed578706621dcc2a55e777e#diff-dfd54bb8cec34d34e6f3c9bab15d76acR127)):

```bash
 ellismg@linux:~/coreclr-demo/packages$ curl -L -O https://nuget.org/nuget.exe
 ```

I would prefer to keep this pull request about the syntax highlighting, not changing the examples themselves.